std/manual.md

```diff
@@ -266,7 +266,7 @@ command line:
 $ deno types
 ```
 
-[This is what the output looks like.](https://gist.github.com/ry/46da4724168cdefa763e13207d27ede5)
+[This is what the output looks like.](https://gist.github.com/trivikr/1d3f6d157b6b311dedd4661ffade6362)
```

Comment:
Question for deno authors: Instead of updating Gist (this one is from https://github.com/denoland/deno/issues/3207#issue-512858920), can you share ideas on how to store output of `deno types` every time a new version is released and link to that instead?

Reply:
You can just link to `cli/js/lib.deno_runtime.d.ts` file. When you run `deno types` it's just outputting contents of that file.

Reply:
As suggested by @bartlomieju
```suggestion
[This is what the output looks like.](https://github.com/denoland/deno/blob/master/cli/js/lib.deno_runtime.d.ts)
```